test/parallel/test-path-makelong.js

```diff
@@ -21,11 +21,12 @@
 
 'use strict';
 const common = require('../common');
+const fixtures = require('../common/fixtures');
 const assert = require('assert');
 const path = require('path');
 
 if (common.isWindows) {
-  const file = path.join(common.fixturesDir, 'a.js');
+  const file = path.join(fixtures.fixturesDir, 'a.js');
```

Comment:
Hey, this can be a `fixures.path('a.js')`

Reply:
I will fix this while landing.